src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -360,7 +360,8 @@ const supplierRequisitionInitialiser = requisition => {
   const route = program ? ROUTES.SUPPLIER_REQUISITION_WITH_PROGRAM : ROUTES.SUPPLIER_REQUISITION;
 
   const sortedData = backingData.sorted('item.name').slice();
-  const filteredData = usingPrograms
+  const isFinalised = requisition.status === 'finalised';
+  const filteredData = !isFinalised
```

Comment:
hmm this does work, but only because if it's not a program there are no items yet! It seems a little odd to me, when you think about non-program requisitions. Just because this is basically ignored, I don't think we should not account for that case explicitly.

We filter when the requisition is NOT finalised AND is a program requisition using this one  : https://github.com/openmsupply/mobile/blob/7c64ed28f73f2be998e2fa6f2dafeaa0798f9941/src/pages/dataTableUtilities/getPageInitialiser.js#L359 

Alternatively you can say: we DONT filter when the requisition is finalised OR is NOT a program requisition

Another one is you could use this:
https://github.com/openmsupply/mobile/blob/7c64ed28f73f2be998e2fa6f2dafeaa0798f9941/src/database/DataTypes/Requisition.js#L68-L70

so you can add to this line:
https://github.com/openmsupply/mobile/blob/7c64ed28f73f2be998e2fa6f2dafeaa0798f9941/src/pages/dataTableUtilities/getPageInitialiser.js#L357

`const { isFinalised, program, items:backingData } = requisition;`

and then you have already `isFinalised` to use :)

Reply:
> Alternatively you can say: we DONT filter when the requisition is finalised OR is NOT a program requisition

Totally agree, I am going to use this one instead, makes this point easier to understand.

> `const { isFinalised, program, items:backingData } = requisition;`
> 
> and then you have already `isFinalised` to use :)

Great, I am using it this way now, thanks for the suggestions!